src/nodes/typeGuards/compound.ts

```diff
@@ -61,6 +61,16 @@ export function isJsxTagNamePropertyAccess(
 	);
 }
 
+/**
+ * @remarks This always returns true.
+ * @see https://github.com/JoshuaKGoldberg/ts-api-utils/pull/16#discussion_r1106579774
+ */
+export function isNamedDeclaration(
+	node: ts.Declaration
+): node is ts.NamedDeclaration {
+	return true;
+}
```

Comment:
@RebeccaStevens I'm up for waiting till #66 is merged before taking in that change here. Up to you?